src/components/cylc/Mutation.vue

```diff
@@ -222,3 +224,9 @@ export default {
   }
 }
 </script>
+
+<style>
```

Comment:
Tip: use [scoped style](https://vuejs.org/api/sfc-css-features.html#scoped-css) to avoid this CSS leaking out where we don't expect it
```suggestion
<style scoped>
```